### PR TITLE
HANDLE AID CLEAR() Produces a Syntax Error in EXEC CICS

### LINKED ISSUE #670: HANDLE AID CLEAR() Produces a Syntax Error in EXEC CICS
## Description ##

A `HANDLE AID CLEAR()` shows a syntax error on parenthesis when it should allow a blank value.

## Steps to reproduce ##
1. Open a COBOL file
2. Add `EXEC CICS HANDLE AID CLEAR() END-EXEC.`
3. See an error on the parenthesis

## Expected behavior ##

No syntax errors are shown.

## Environment ##
 - OS: All
 - IDE: All

## Additional information ##

Seems like a blank value is allowed only for resetting operations, i.e. only for CLEAR

### PR DESCRIPTION
**Pull request by grianbrcom**
Closes #670

Signed-off-by: Anton Grigorev <anton.grigorev@broadcom.com>

### CODE REVIEW

File: server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExecCicsHandleAidClear.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package com.broadcom.lsp.cobol.usecases;
+
+import com.broadcom.lsp.cobol.usecases.engine.UseCaseEngine;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This test check the positive scenarios of using CLEAR in HANDLE AID.
```
Comment (temanbrcom) [inline server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExecCicsHandleAidClear.java:24]: "checks"

**Review by temanbrcom — APPROVED** [review]
📲

File: server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExecCicsHandleAidClear.java (hunk 1 above)
Comment (grianbrcom) [reply server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExecCicsHandleAidClear.java:24]: Right.

**Comment by temanbrcom** [comment]
@asatklichov 
@ap891843

**Review by asatklichov — APPROVED** [review]

File: server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExecCicsHandleAidClear.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This test checks the positive scenarios of using CLEAR in HANDLE AID.
+ * The valid usages are: CLEAR() and CLEAR(cics_label).
+ */
+public class TestExecCicsHandleAidClear {
```
Comment (temanbrcom) [inline server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExecCicsHandleAidClear.java:27]: Should not be public

**Comment by ap891843** [comment]
@temanbrcom Should empty_parens be allowed for all options of HANDLE AID or its just valid for CLEAR?

**Review by ap891843 — APPROVED** [review]

**Pull request merged**